### PR TITLE
Additional example

### PR DESCRIPTION
**Pull request by Brianfit**
Using the sensor as a trigger. The syntax for this took me ages to work out, be great to save someone else the hours.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/sensor.speedtest.markdown [hunk 1]
```diff
@@ -90,6 +90,24 @@ sensor:
...
+ - alias: 'Internet Speed Glow Connect Great' 
+    trigger: 
+      platform: template
+      value_template: '{{ states.sensor.speedtest_download.state|float > 10}}' 
+    action:      
+      service: shell_command.green
+  - alias: 'Internet Speed Glow Connect Poor' 
+    trigger: 
+      platform: template
+      value_template: '{{ states.sensor.speedtest_download.state| float < 10 }}'
```
Comment (fabaff) [inline source/_components/sensor.speedtest.markdown]: Dito

File: source/_components/sensor.speedtest.markdown [hunk 2]
```diff
@@ -90,6 +90,24 @@ sensor:
...
 ```
 
+### {% linkable_title Using as a trigger for another action %}
+
+```yaml
+# Example configuration.yaml entry
+ - alias: 'Internet Speed Glow Connect Great' 
+    trigger: 
+      platform: template
+      value_template: '{{ states.sensor.speedtest_download.state|float > 10}}'
```
Comment (fabaff) [inline source/_components/sensor.speedtest.markdown]: To avoid render of the template, use `{% raw %}{{your_template}}{% endraw %}`.

Comment (Brianfit) [reply source/_components/sensor.speedtest.markdown]: Thanks @fabaff. Not 100% sure I've got the syntax right -- is there a way to preview changes that renders the page as the Homeassistant site does? Preview here isn't that useful for seeing the result.

**Comment by frenck** [comment]
Great first-time contribution to our documentation @Brianfit! 🥇

**Pull request merged**